crates/model/src/order.rs

```diff
@@ -413,18 +413,41 @@ impl OrderCreation {
     }
 
     /// Recovers the owner address for the specified domain, and then verifies
-    /// it matches the expected address.
+    /// it matches the expected addresses.
+    ///
+    /// Expected addresses can come from two sources: directly from the order
+    /// creation and as part of the calldata.
```

Comment:
Do you mean `appdata` here?
```suggestion
    /// creation and as part of the appdata.
```